types/jsdom.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'jsdom';
```

Comment:
Have you tried installing the package types definition?
`npm -i --save-dev @types/jsdom`

If they are available, then we do not need this file. If they are not available, then we should declare the package here.